releasenotes/notes/graphql-error-span-events-add-extensions-5eece423cc8ff93e.yaml

```
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    graphql: Add user provided extensions to graphql error span events through the environment variable `DD_TRACE_GRAPHQL_ERROR_EXTENSIONS`.
```

Can we simplify this release note? It's a bit difficult to understand the value of this change

```suggestion
    graphql: Adds user provided extensions to graphql error span events through the environment variable `DD_TRACE_GRAPHQL_ERROR_EXTENSIONS`.
```

---

@mabdinur I modified to try to make it more clear, let me know if you think it should be improved 👍